src/App.test.jsx

```diff
@@ -0,0 +1,30 @@
+import { render, screen, waitForElementToBeRemoved } from "@testing-library/react"
+import { MemoryRouter } from "react-router-dom";
+import userEvent from '@testing-library/user-event';
+import App from './App';
+
+it('should get a list of rick and morty characters', async () => {
+    render(
+    <MemoryRouter>
+      <App />
+    </MemoryRouter>
+    )
+
+    const loading = screen.getByText(/Loading Pickle Rick/i);
+    await waitForElementToBeRemoved(loading);
+    await screen.findByText('Morty Smith');
```

Comment:
If you are rendering a list I would expect to see some expectations that the screen contains more than one element.